bigtest/network/factories/package.js

```diff
@@ -86,5 +107,13 @@ export default Factory.extend({
       let visibilityData = server.create('visibility-data');
       packageObj.update('visibilityData', visibilityData.toJSON());
     }
+    if (!packageObj.proxy) {
+      let proxy = server.create('proxy', {
+        inherited: false,
+        id: 'bigTestJS'
+      });
+      packageObj.update('proxy', proxy.toJSON());
+      packageObj.save();
+    }
```

Comment:
There is a naming conflict here too, but your is better. So, we'll leave it this way and I'll adjust mine after you merge!